### ISSUE TITLE
Get one frame with a lot of frame in it ?

### ISSUE THREAD

**Issue by shankiflang** [PHP-FFMpeg/PHP-FFMpeg#633]
| Q              | A
| -------------- | ---
| Bug?           | no
| New Feature?   | maybe
| Version Used   | ffmpeg-20190225-f948082-win64-static
| FFmpeg Version | FFmpeg or AVConv and version
| OS             | Windows

Hey,
I juste want to know if it's possible to get one frame with a lot of frame in it ? like with this command :
`ffmpeg -loglevel panic -y -i "video.mp4" -frames 1 -q:v 1 -vf "select=not(mod(n\,40)),scale=-1:120,tile=100x1" video_preview.jpg` ?

Regards

**Comment by jens1o** [comment]
No, this is not yet possible, although I find it an very interesting approach, if we could make it more dynamic.
